modules/storage/storage-account/README.md

```diff
@@ -83,25 +75,101 @@ output storageAccountID string = storageAccount.outputs.id
 
 ### Example 2
 
-This example creates a new Storage Account with the name "mystorageaccount" in the "myresourcegroup" resource group located in the East US region. The Storage Account is configured to use zone-redundancy and virtual network access. The module output is the ID of the created or existing Storage Account, which can be used in other Azure resource deployments.
-
 ```bicep
-param location string = 'eastus'
-param name string = 'mystorageaccount'
-param newOrExisting string = 'new'
-param resourceGroupName string = 'myresourcegroup'
-param isZoneRedundant bool = true
-
-module storageAccount 'br/public:storage/storage-account:2.0.2' = {
-  name: 'mystorageaccount'
+module test '../main.bicep' = {
```

Comment:
It seems the example was copied from a test. Can you update the path? You may also need to add `prereq` to the example since it's a dependency of the module.

Reply:
ok

Reply:
updated